src/components/ActionLink/ActionLink.types.ts

```diff
@@ -16,7 +16,12 @@ export interface BaseActionLinkProps extends BaseComponentProps {
   /**
    * The visible text.
    */
-  text: string;
+  text: string | React.ReactNode;
```

Comment:
Should this remain only string? it looks like you solved this with only string and we probably don't want to allow arbitrary markup here?